op-chain-ops/genesis/config.go

```diff
@@ -64,6 +64,8 @@ type DeployConfig struct {
 	// BatchSenderAddress represents the initial sequencer account that authorizes batches.
 	// Transactions sent from this account to the batch inbox address are considered valid.
 	BatchSenderAddress common.Address `json:"batchSenderAddress"`
+	// HotShot Contract address
+	HotShotContractAddress common.Address `json:"hotshotContractAddress"`
```

Comment:
I wonder if this should be a pointer with `json:"omitempty"`, so that it is optional when running in legacy mode

Reply:
I think we will also need to add this to `devnetL1-espresso.json` so it gets set in `init.go`